### ISSUE TITLE
Safe Path of Movement

### ISSUE THREAD

**Issue by rsngamt** [nic-delhi/AarogyaSetu_Android#33]
With easing of lockdown Movement of people will increase so an navigation option should be added so that people can move from one location to another location 
by safe (green) path (Contamination free or low risk) instead of people relying on Google Maps or for shortest route between two points

**Comment by akx** [comment]
Wouldn't those safe paths then become unsafe, as people including asymptomatic carriers begin using them more or less exclusively?

**Comment by HaneetGH** [comment]
ask #57 I've suggested..

App just suggest a route which is safe..

all other navigation and stuff still be happen on google maps

**Comment by HaneetGH** [comment]
> Wouldn't those safe paths then become unsafe, as people including asymptomatic carriers begin using them more or less exclusively?

yeah.. but if this feature is only available for people with no covid19

**Comment by aravindvnair99** [comment]
@rsngamt Please check for existing issues before opening a new one. You can close this and proceed there. Duplicate of #57

**Comment by safiyat** [comment]
@aravindvnair99 It is funny because #33 comes before #57.

**Comment by aravindvnair99** [comment]
> @aravindvnair99 It is funny because #33 comes before #57.

@safiyat Right, my bad. Accidently tagged the wrong one. I have deleted my comment.
